src/ProgressOnderwijsUtils/Data/ParameterizedSqlExtensions.cs

```diff
@@ -45,4 +45,11 @@ public static ParameterizedSql AsUnrolledSqlInExpression<T>(this IEnumerable<T>
             0 => SQL($"(null)"),
             _ => SQL($"(").Append(set.Select(item => SQL($"{AsSqlParam(item)}")).ConcatenateSql(SQL($","))).Append(SQL($")")),
         };
+
+    /// <summary>
+    /// Generate an unique name which can be used safely as an alias for the given query.
+    /// </summary>
+    [Pure]
+    public static ParameterizedSql GenerateUniqueQueryAlias(this ParameterizedSql query)
```

Comment:
Ik denk dat deze extension method beetje overdreven is gezien hoe niche de use-case is.

Reply:
evt. als non-ext method naast `GenerateUniqueQueryAlias` zetten?